flight/targets/pipxtreme/fw/pios_board.c

```diff
@@ -162,12 +162,14 @@ void PIOS_Board_Init(void)
 
 	ModuleSettingsSet(&moduleSettings);
 
+#ifdef PIOS_INCLUDE_OPENLRS
```

Comment:
Is the PipX useful for anything without olrs? It would seem to not be.

Reply:
Not really, but I turned it off during re-bringup etc.